docs/content/concepts/ops-jobs-graphs/ops.mdx

```diff
@@ -1,42 +1,41 @@
-| <PyObject object="op" decorator />      | The decorator used to define ops. The decorated function is called the `compute_fn`. The decorator returns an <PyObject object="OpDefinition" />.                                               |
-| <PyObject object="In" />                | Ins define the inputs to an op compute function. These are defined on the `ins` argument to the <PyObject object="op" decorator/> decorator.                                                    |
-| <PyObject object="Out" />               | Outs define the outputs of an op compute function. These are defined on the `outs` argument to the <PyObject object="op" displayText="op" decorator/> decorator.                                |
-| <PyObject object="OpExecutionContext"/> | An optional first argument to a op's compute function, providing access to Dagster system information such as resources, logging, and the underlying Dagster instance.                          |
-| <PyObject object="OpDefinition"  />     | Class for ops. You almost never want to use initialize this class directly. Instead, you should use the <PyObject object="op" decorator /> which returns an <PyObject object="OpDefinition" />. |
+| <PyObject object="op" decorator />      | A decorator used to define ops. Returns an <PyObject object="OpDefinition" />. The decorated function is called the "compute function".                                              |
+| <PyObject object="In" />                | An input to to an op. Defined on the `ins` argument to the <PyObject object="op" decorator/> decorator.                                                    |
+| <PyObject object="Out" />               | An output of an op. Defined on the `out` argument to the <PyObject object="op" displayText="op" decorator/> decorator.                                |
+| <PyObject object="OpExecutionContext"/> | An object exposing Dagster system APIs for resource access, logging, and more. Can be injected into an op by specifying `context` as the first argument of the compute function.                           |
+| <PyObject object="OpDefinition"  />     | Class for ops. You will rarely want to instantiate this class directly. Instead, you should use the <PyObject object="op" decorator />. |
 
 ## Overview
 
-Ops are used to define computations. Ops can later be assembled into [Jobs](/concepts/ops-jobs-graphs/jobs-graphs). Ops generally perform one specific action and are used for batch computations. For example, you can use an op to:
+Ops are Dagster's atomic unit of computation. Individual ops should perform relatively simple tasks. Collections of ops can then be assembled into [Jobs or Graphs](/concepts/ops-jobs-graphs/jobs-graphs) to perform more complex tasks. Some examples of tasks appropriate for a single op:
 
-- Derive a data set from some other data sets.
```

Comment:
Any particular reason to take this one out? IMO, this is the top task for an op.

Reply:
My reason for this change is that "derive a dataset from another dataset" is vague wrt scale. This could describe an op, or an entire pipeline, or something even larger.

Reply:
I like keeping this one. Agreed with Sean's assessment but because of the same reason, I like the ambiguity here to show that an op the where you process a data set.